static/app/components/assigneeBadge.stories.tsx

```diff
@@ -39,10 +40,30 @@ export default storyBook(AssigneeBadge, story => {
     const [chevron1Toggle, setChevron1Toggle] = useState<'up' | 'down'>('down');
     const [chevron2Toggle, setChevron2Toggle] = useState<'up' | 'down'>('down');
 
+    const team: Team =
+      teams.length > 0
```

Comment:
```suggestion
      teams.length > 0
```
! NIT ! can just do `teams.length` here!